server/rollback_runner.go

```diff
@@ -0,0 +1,154 @@
+package server
+
+import (
+	"context"
+	"encoding/json"
+	"sync"
+
+	"github.com/pkg/errors"
+	"go.uber.org/zap"
+
+	"github.com/bytebase/bytebase/api"
+	"github.com/bytebase/bytebase/common/log"
+	"github.com/bytebase/bytebase/plugin/db"
+	"github.com/bytebase/bytebase/plugin/db/mysql"
+)
+
+const (
+	rollbackSQLChanLen = 100
+)
+
+var (
+	generateRollbackSQLChan = make(chan *api.Task, rollbackSQLChanLen)
+)
+
+// NewRollbackRunner creates a new rollback runner.
+func NewRollbackRunner(server *Server) *RollbackRunner {
+	return &RollbackRunner{
+		server: server,
+	}
+}
+
+// RollbackRunner is the rollback runner generating rollback SQL statements.
+type RollbackRunner struct {
+	server *Server
+}
+
+// Run starts the rollback runner.
+func (r *RollbackRunner) Run(ctx context.Context, wg *sync.WaitGroup) {
+	defer wg.Done()
+	r.retryGetRollbackSQL(ctx)
+	for {
+		select {
+		case task := <-generateRollbackSQLChan:
+			r.getRollbackSQL(ctx, task)
```

Comment:
The name should not be getRollbackSQL() any more if we look at its implementation. GenerateRollbackSQL or BackfillRollbackSQL.